src/main/java/frc/robot/RobotContainer.java

```diff
@@ -52,11 +58,25 @@ public RobotContainer() {
     }
 
     private void configureDefaultCommands() {
-        shooter.setDefaultCommand(new Shoot(shooter, () -> 4));
-        conveyor.setDefaultCommand(new ConveyorDefaultCommand(conveyor, Constants.Conveyor.POWER));
     }
 
     private void configureButtonBindings() {
+        DoubleSupplier distanceFromTarget = () -> visionModule.getDistance().getAsDouble();
```

Comment:
```suggestion
        DoubleSupplier distanceFromTarget = () -> visionModule.getDistance().orElse(0);
```
it's safer this way.